frontend/src/app/modules/landing/registration/registration.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="registration-container">
+    <div class="registration-content">
+        <h4>Sing Up</h4>
+
+        <div class="additional-sing-up">
+            <button>Sing Up with GitHub</button>
```

Comment:
Sign `in` with ...

Reply:
not sure, but probably some differences are

Reply:
Authentification is quite similar (especially from user perspective), but it will be more understandable to have sign up here

Reply:
By the way, I think we can call it the same for login and registration, becouse it's allow us to create one component for additional authentication and reused in login/registration components, we can name for example "Continue with Google" and also it will make our  code more single responsibility. What do you think?